src/RazorSdk/Tasks/StaticWebAssets/GenerateStaticAssetsManifest.cs

```diff
@@ -0,0 +1,157 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text.Encodings.Web;
+using System.Text.Json;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+
+namespace Microsoft.AspNetCore.Razor.Tasks
+{
+    public class GenerateStaticWebAssetsManifest : Task
+    {
+        // Since the manifest is only used at development time, it's ok for it to use the relaxed
```

Comment:
It's not because it's dev time, but more so because it's not JSON that's emitted inside HTML.